ui/src/app/pages/admin/admin-user-access.tsx

```diff
@@ -36,7 +36,9 @@ export const AdminUserAccess = () => {
   );
 
   const parseUserEmailInput = (input: string) => {
-    setUserEmails(input.split(/[,\n]+/).map((email) => email.trim()));
```

Comment:
Have you tried `input?.split` instead?  That will protect against null and undefined.

Reply:
good idea